src/menu/src/MenuItem.js

```diff
@@ -122,7 +147,12 @@ MenuItem.propTypes = {
   /**
    * Callback to invoke onkeypress
    */
-  onKeyPress: PropTypes.func
+  onKeyPress: PropTypes.func,
+
+  /**
+   * Flag to indicate whether the menu item is disabled or not
+   */
+  disabled: PropTypes.bool
```

Comment:
Can you update the typedefs as well in `index.d.ts`?

Reply:
Done ✅